### ISSUE TITLE
Asset compilation fails on Windows 11 after upgrade to 2.1.0

### ISSUE THREAD

**Issue by amadeann** [tighten/laravel-mix-jigsaw#27]
Reproduction:

Follow the installation guide from https://jigsaw.tighten.com/docs/installation/. The installation fails at step 3: `vendor/bin/jigsaw init`.

Running `npm run dev` after an upgrade to v2.1.0 from v2.0.0 returns the following error:

```
webpack compiled successfully
events.js:377
      throw er; // Unhandled 'error' event
      ^

Error: spawn vendor\bin\jigsaw ENOENT
    at Process.ChildProcess._handle.onexit (internal/child_process.js:277:19)
    at onErrorNT (internal/child_process.js:472:16)
    at processTicksAndRejections (internal/process/task_queues.js:82:21)
Emitted 'error' event on ChildProcess instance at:
    at Process.ChildProcess._handle.onexit (internal/child_process.js:283:12)
    at onErrorNT (internal/child_process.js:472:16)
    at processTicksAndRejections (internal/process/task_queues.js:82:21) {
  errno: -4058,
  code: 'ENOENT',
  syscall: 'spawn vendor\\bin\\jigsaw',
  path: 'vendor\\bin\\jigsaw',
  spawnargs: [ 'build', 'local' ]
}
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @ dev: `mix`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the @ dev script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

**Downgrading to 2.0.0 fixes the issue.**

I tried running the command both from Powershell (as regular user, then as administrator), ~~and running it from within WSL~~, and got the same error message in all cases.

I also tried deleting node_modules and npm cache, and reinstalling them.

**Comment by amadeann** [comment]
Adding `shell: true` attribute to `spawn` call ([see docs](https://nodejs.org/docs/latest-v14.x/api/child_process.html#child_process_child_process_spawnsync_command_args_options)) fixes it for me. Default value is `false`. I'm not the best with Node, so it would be great if someone could confirm if adding this doesn't mess things up on other platforms.

```
const jigsaw = () => spawn(bin, ['build', env], { stdio: 'inherit', shell: true }).on('exit', (code) => {
    if (code > 0) {
        console.warn(`\nJigsaw build failed, see above.`);
    }
});
```

**Comment by bakerkretzmar** [comment]
Interesting. I'll test on macOS this week, I can't see why adding `shell` would be an issue. Thanks.

**Comment by bakerkretzmar** [comment]
This was with a newly installed Jigsaw site, right? I'm really confused about why it would fail in WSL too, that's basically a linux environment I thought? I'm happy to set `shell` to `true` I just want to understand what's happening. Can you try to see if `process.env.ComSpec` is defined inside the Node process? Is the error output you shared above from the run in WSL, and if not can you share that? In WSL is `/bin/sh` in your path? Thanks.

**Comment by amadeann** [comment]
@bakerkretzmar 
I double checked it now with WSL, and actually it works fine, both with my existing and newly created projects. Sorry about the confusion on this point!

The problem is only with Windows installs. `process.env.ComSpec` returns `C:\WINDOWS\system32\cmd.exe`.

**Comment by bakerkretzmar** [comment]
Fixed in v2.1.1.
